crates/subspace-networking/src/node_runner.rs

```diff
@@ -291,7 +291,9 @@ impl NodeRunner {
                 .iter()
                 .any(|protocol| protocol.as_bytes() == RELAY_HOP_PROTOCOL_NAME);
 
-            let proper_protocols_supported = !relay_server_enabled && kademlia_enabled;
+            // TODO: figure out how to filter peers by supported protocols
```

Comment:
We merged https://github.com/subspace/subspace/pull/762 to address that.

Reply:
I don't see rolling it back. We introduced bootstrap-node app to address this issue, you can pass its address to the internal networking instances and have a connected network.

Reply:
I switch request to `PeerInfo` protocol, this should handle the job right?

Reply:
It will only replace dummy requests, however you should restore the original relay-node filtering and start bootstrap-node app for testing.

Reply:
Oh, gotcha. Let me revert just relay node filtering.